osclib/cache.py

```diff
@@ -90,7 +90,7 @@ class Cache(object):
         '/source/([^/]+)/_meta$': TTL_LONG,
         '/source/([^/]+)/(?:[^/]+)/(?:_meta|_link)$': TTL_LONG,
         '/source/([^/]+)/dashboard/[^/]+': TTL_LONG,
-        '/source/([^/]+)/_attribute/[^/]+': TTL_MEDIUM,
+        '/source/([^/]+)/_attribute/[^/]+': TTL_SHORT,
```

Comment:
Not particularly related.

Reply:
Nothing wrong with the change. Just not related to commit "Simplify the repo_checker handling overlays".